projects/kit/components/input-number/input-number.component.ts

```diff
@@ -82,6 +86,10 @@ export class TuiInputNumberComponent
 
     protected readonly options = inject(TUI_INPUT_NUMBER_OPTIONS);
 
+    protected get numberFormat(): TuiNumberFormatSettings {
```

Comment:
I think we should just add subscription, instead of trying to turn it into `BehaviorSubject` with `.value`. With `tuiWatch`.